webui/react/src/e2e/tests/experimentList.spec.ts

```diff
@@ -28,6 +29,13 @@ test.describe('Experiment List', () => {
             `experiment create ${fullPath('examples/tutorials/mnist_pytorch/adaptive.yaml')} --paused --project_id ${newProject.response.project.id}`,
           );
         });
+      const experiments: ExperimentBase[] = JSON.parse(
+        detExecSync(
+          `project list-experiments --json ${newWorkspace.response.workspace.name} ${newProject.response.project.name}`,
+        ),
+      );
+      detExecSync(`experiment kill ${experiments[experiments.length - 1]?.id}`);
```

Comment:
does this interfere with any of the other tests? should you be creating a second experiment for this scenario? or maybe put this archive logic in a separate describe?